### ISSUE TITLE
Clicking question mark for help menu blurs whole page

### ISSUE THREAD

**Issue by jsha** [rust-lang/docs.rs#1076]
Steps to reproduce:

1. Visit a docs page, like https://docs.rs/ureq/1.5.1/ureq/
2. Click the '?' icon next to the search box, OR press ? on the keyboard.

(On Chrome 85.0.4183.121 (Official Build) (64-bit), Linux)

Expected result:

Help menu pops up.

Actual result:

Help menu pops up, but it is blurred along with the whole page.

![image](https://user-images.githubusercontent.com/220205/95285478-bfcab380-0815-11eb-80e8-29939019cf19.png)

**Comment by jyn514** [comment]
Rustdoc must have recently changed its styles. It's applying blur to `body`, not just to the `<section id=main>`. @GuillaumeGomez can you warn docs.rs before making changes like this?

There are two possible solutions:
- Fix it upstream in rustdoc, which I prefer. This has the downside of #464.
- Mangle the styles further as they come through S3: https://github.com/rust-lang/docs.rs/blob/1fb61ec75249bb95fffb64b63ff739de35f548b9/src/utils/html.rs#L40
Remove the style from `<body>` and add it to `<section>`; add CSS for blurring `section`. Rustdoc's won't work because it only applies to top-level children of `body`.

```css
body.blur > :not(#help) {
    filter: blur(8px);
    -webkit-filter: blur(8px);
    opacity: .7;
}
```

**Comment by GuillaumeGomez** [comment]
It's been like this for ages, no?

**Comment by benediktwerner** [comment]
This was fixed by #1080
